Projects/App/Sources/UI/Place/AddPlace/AddPlaceResultViewController.swift

```diff
@@ -0,0 +1,270 @@
+        $0.text = "만든사람"
+        $0.textColor = .zestyColor(.gray3C)
+        $0.font = .systemFont(ofSize: isSE ? 13 : 16, weight: .medium)
+        return $0
+    }(UILabel())
+    
+    private lazy var dateLabel: UILabel = {
+        $0.text = "YYYY.MM.DD"
+        $0.textColor = .zestyColor(.gray3C)
+        $0.font = .systemFont(ofSize: isSE ? 13 : 16, weight: .medium)
+        return $0
+    }(UILabel())
+    
+    private lazy var saveButton: UIButton = {
+        let config = UIImage.SymbolConfiguration(paletteColors: [.zestyColor(.point) ?? .red])
+        let downloadImage = UIImage(systemName: "square.and.arrow.down", withConfiguration: config)
+        $0.setImage(downloadImage, for: .normal)
+        $0.setTitle(" 리뷰 카드 저장하기", for: .normal)
+        $0.setTitleColor(.zestyColor(.point), for: .normal)
+        $0.titleLabel?.font = UIFont.systemFont(ofSize: 13, weight: .medium)
+        $0.addTarget(self, action: #selector(saveButtonDidTap), for: .touchUpInside)
+        return $0
+    }(UIButton())
+    
+    private lazy var doneButton: UIButton = {
+        $0.configuration = .borderedTinted()
+        $0.setTitle("완료", for: .normal)
+        $0.addTarget(self, action: #selector(doneButtonDidTap), for: .touchUpInside)
+        return $0
+    }(UIButton())
+    
+    // MARK: - LifeCycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+        createLayout()
+        setNavigationBar()
+    }
+    
+    // MARK: - Function
+    @objc func backButtonDidTap() {
+        self.navigationController?.popViewController(animated: true)
+    }
+    
+    @objc func saveButtonDidTap() {
+        // ShareSheet
+    }
+    
+    @objc func doneButtonDidTap() {
+        self.navigationController?.popToRootViewController(animated: true)
+    }
+    
+}
+
+// MARK: - Binding
+extension AddPlaceResultViewController {
+    
+    private func bind() {
+        
+    }
+    
+}
+
+// MARK: - UI Function
+
+extension AddPlaceResultViewController {
+    
+    private func configureUI() {
+        view.backgroundColor = .white // zestyColor(.backgroundColor)
+        
+        iconView.kf.setImage(with: URL(string:  "https://user-images.githubusercontent.com/63157395/197410857-e13c1bbb-b19a-4c59-a493-77501a4a529b.png"))
+    }
+    
+    private func createLayout() {
+        view.addSubviews([titleView, ticketImageView, iconView,
+                          categoryTagLabel, placeNameLabel, addressLabel,
+                          orgTitle, creatorTitle, dateTitle,
+                          orgLabel, creatorLabel, dateLabel,
+                          saveButton, doneButton])
+        
+        titleView.snp.makeConstraints {
+            $0.top.equalTo(view.safeAreaLayoutGuide)
+            $0.horizontalEdges.equalToSuperview().inset(20)
+        }
+        
+        ticketImageView.snp.makeConstraints {
+            $0.top.equalTo(titleView.snp.bottom).multipliedBy(1.035)
```

Comment:
아 피그마 확인했는데 티켓부분 한 번 대공사가 필요하겠네요...
일단 빠른 출시가 중요하니까 여기 레이아웃은 그대로 두는걸로 합시다!

그런데 MainTitleView랑 ticketImageView 사이 간격이 너무 좁아서 여기만 .offset(20)으로 고쳐주세요.
```suggestion
            $0.top.equalTo(titleView.snp.bottom).offset(20)
```

Reply:
우선 바꿨으나,, 타이틀뷰 고치는게 시급함 !

Reply:
a11048b52347cbf79d7ac7fd4d9c723f97516a4a